### PR TITLE
Change some design and default values

### PR DESCRIPTION
**Pull request by anddea**
Changed some overlay icons
## New (also new SB icon near gear icon)
ic_new

## Old
ic_old

## SponsorBlock New Segment menu
sb_new

## Old SB new segment icons
sb old

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/youtube/utils/sponsorblock/SponsorBlockPatch.kt [hunk 1]
```diff
@@ -62,6 +62,9 @@ object SponsorBlockPatch : ResourcePatch() {
                 "ic_sb_edit.xml",
                 "ic_sb_logo.xml",
                 "ic_sb_publish.xml",
+ "ns_bg.xml",
```
Comment (YT-Advanced) [inline src/main/kotlin/app/revanced/patches/youtube/utils/sponsorblock/SponsorBlockPatch.kt]: You should make it align like this
```suggestion
                "ns_bg.xml",
                "ic_sb_forward.xml",
                "ic_sb_backward.xml",
```

Comment (anddea) [reply src/main/kotlin/app/revanced/patches/youtube/utils/sponsorblock/SponsorBlockPatch.kt]: Sure, I just automated it, that's why it got that way. Fixed it.

**Comment by YT-Advanced** [comment]
Also you just update overlay icon, so can u  send me a screenshot before and after pls

**Comment by anddea** [comment]
> Also you just update overlay icon, so can u send me a screenshot before and after pls

Added to original post

**Comment by Mohammed1520** [comment]
Also if you could please change download icon and fullscreen icon to be matched with official Revanced. As these ones of RVX are bigger than Revanced 
Screenshot_20231025_122205_app rvx android youtube_edit_1685871087392750

**Comment by YT-Advanced** [comment]
These icon look nice

**Comment by anddea** [comment]
> to be matched with official Revanced

ReVanced uses SVG overlay icons which is better as it's vector image format and RvX uses png files, I can spend some time on returning it back to SVG if @inotia00 will allow it. But maybe icons I made may suit you better. Btw SB icons are still SVG and they have better quality. But in real life use case you won't see quality issues unless you have very big high quality display.

Icons used right now in RvX releases are bold (too bright), kinda ugly and out of place IMO (ReVanced icons too), so I made these icons as I find it more beautiful (actually I made them months ago and wanted to refine it but I'm lazy) .



> These icon look nice

Which ones? That I made or existing ones?

**Comment by Francesco146** [comment]
now that someone mentioned it, icons added in overlay have a delay when, either full screen mode or not, the player controls are deactivated. The official icons all disappear in unison, however the added overlay icons lag a bit, around 10ms to 15ms.

for better understanding: 


https://github.com/inotia00/revanced-patches/assets/49027005/65918f24-6b9c-4952-8b74-264bcdb095cf

**Comment by YT-Advanced** [comment]
> Which ones? That I made or existing ones?

New ones

**Comment by anddea** [comment]
> The official icons all disappear in unison, however the added overlay icons lag a bit, around 10ms to 15ms.

Very old problem. Including SB icons. Probably it has to do with adding new resource files through patching, because if we replace existing icons (e.g. fullscreen icon), then fading in/out is not delayed.

**Comment by inotia00** [comment]
It seems more appropriate to allow the user to select individual icons for overlay buttons through `options.json`

**Comment by inotia00** [comment]
Patch will be rebased to support revanced-patcher 19.0.0

I don't want PR branches to conflict, so instead of waiting for contributors to reflect change requests, I'll merge them first and apply the changes to the rebase process

**Comment by anddea** [comment]
> It seems more appropriate to allow the user to select individual icons for overlay buttons through `options.json`

I think most of the people don't even know about existence of options.json, and currently there's no way to select wanted icons afaik. Not many people will know about customization options and even less will spend time on things like customization, that's why defaults matter. The best way to give users ability to change icons is through UI instead of options.json. 

Reconsider these changes as they make app look better overall (in integrations repo as well), that will increase UX. I personally know a lot of people who won't use some features just because they're unsatisfied with the look.

**Comment by inotia00** [comment]
no need to worry about that
most users use RVX Manager
and in the next version of RVX Manager, it will be easier to select the patch option as shown in the attached screenshot
(just switch the ON/OFF toggle)

Options png

**Comment by anddea** [comment]
What about other defaults, paddings and font size that looks better, default values for feed flyout filter patch (download and queue buttons) that work only with prenium? [Integrations PR](https://github.com/inotia00/revanced-integrations/pull/26) has nothing to do with icons

**Comment by MondayNitro** [comment]
> What about other defaults, paddings and font size that looks better, default values for feed flyout filter patch (download and queue buttons) that work only with prenium? [Integrations PR](https://github.com/inotia00/revanced-integrations/pull/26) has nothing to do with icons

Sir, if you don't mind, can you also try to create outline icons for shorts player controls, which inotia can merge in rvx?
Screenshot_20231028-140032_Trebuchet~2

**Comment by anddea** [comment]
> Sir, if you don't mind, can you also try to create outline icons for shorts player controls, which inotia can merge in rvx? 

Sure. Will do it in my spare time (after 20-30 hrs).

**Comment by anddea** [comment]
https://github.com/inotia00/revanced-patches/pull/30

**Pull request merged**